checks/evaluation/finding.go

```diff
@@ -29,3 +29,15 @@ func nonNegativeFindings(findings []finding.Finding) []finding.Finding {
 	}
 	return ff
 }
+
+func negativeFindings(findings []finding.Finding) []finding.Finding {
+	var ff []finding.Finding
+	for i := range findings {
+		f := &findings[i]
+		if f.Outcome == finding.OutcomePositive {
```

Comment:
nit: the function is called `negativeFindings` but it's looking for non-positive findings. Can we have the name match the behavior (or vice-versa)

Reply:
I don't think naming it `positiveFindings` addressed this. As it skips over `positiveFindings`

Reply:
Removed it altogether. AFAICS, this does not exist in the current evaluation either: https://github.com/ossf/scorecard/blob/74c57cd1bc54d643a5e77fa4c3c3c2c1309ff84f/checks/evaluation/contributors.go#L72